### PR TITLE
Feature/LEAF 1828/vue header (new)

### PR DESCRIPTION
**Pull request by aerinkayne**
Header based on Jira ticket 1828.  Still in progress, but general feedback welcome.
(imo I don't think having the nav links on the left side, per the ticket, is better.  The menus are in the way, and people are used to them being elsewhere)

### CODE REVIEW

**Review by AmandaSumner — APPROVED** [review]

**Review by Pelentan — APPROVED** [review]
Looks good.

**Review by raphaelfretter — APPROVED** [review]

File: LEAF_Request_Portal/admin/index.php [hunk 1]
```diff
@@ -54,10 +54,11 @@
 }
 
 $main = new Smarty;
-$t_login = new Smarty;
-$t_menu = new Smarty;
-$o_login = '';
-$o_menu = '';
+//nav (menu) is handled with vue component.  login does not look like it is still used
```
Comment (gsteinhoff) [inline LEAF_Request_Portal/admin/index.php:57]: Are we removing these commented out code?  We shouldn't leave it in if no longer used.

File: LEAF_Request_Portal/admin/templates/main.tpl [hunk 2]
```diff
@@ -40,43 +43,76 @@
...
+                    <admin-leaf-nav :inner-width="windowInnerWidth"
+                                    orgchart-path='{json_encode($orgchartPath)}'
+                                    site-type='{json_encode($siteType)}'></admin-leaf-nav>
+                </ul>
+                <ul id="nav-user-info">
+                    <leaf-user-info :inner-width="windowInnerWidth" user-name='{json_encode($name)}'></leaf-user-info>
+                </ul>
+            </nav>
+        </header>
+        {*
```
Comment (gsteinhoff) [inline LEAF_Request_Portal/admin/templates/main.tpl:81]: Again, we should remove old code if no longer using it.

File: LEAF_Request_Portal/admin/templates/vue_components/leaf_header/vue_leaf_header.js [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+            type: String,
+            required: false,
+            default: 'rgb(255,255,255)'
+        }
+    },
+    template:
+        `<p v-if="leafSecure==='0'" id="scrolling-leaf-warning" :style="{backgroundColor: bgColor, color: textColor}"><slot></slot></p>`
+});
+
+//site info  ISSUE not currently used
```
Comment (gsteinhoff) [inline LEAF_Request_Portal/admin/templates/vue_components/leaf_header/vue_leaf_header.js:100]: again commented code should be removed.

**Review by gsteinhoff — CHANGES_REQUESTED** [review]
The only issue I have with this is leaving in the commented-out code.  These should be removed as they can be retrieved in Github as needed for any reason.  Rest of it looks really good.

**Review by gerald-todd — APPROVED** [review]
Looks good

**Review by gsteinhoff — APPROVED** [review]
thanks for taking that commented code out

**Pull request closed without merging**